### PR TITLE
create builds table

### PR DESCRIPTION
**Pull request by pytlesk4**
[builds model](https://next.stoplight.io/stoplight/next-api/blob/master/builds.oas2.yml?view=/models/builds-output)

### CODE REVIEW

File: db/migrate/20180508162616_create_builds.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  def up
+    create_table :builds do |t|
+      t.integer :namespace_id
+      t.integer :project_id
+      t.integer :active_build_id
+      t.string :status
+      t.string :file_path
+      t.string :comment
+      t.string :commmit
```
Comment (rossmcdonald) [inline db/migrate/20180508162616_create_builds.rb]: Typo

Comment (pytlesk4) [reply db/migrate/20180508162616_create_builds.rb]: fixed! Good catch

File: db/migrate/20180508162616_create_builds.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      t.string :branch
+      t.datetime :ts, null: false
```
Comment (rossmcdonald) [inline db/migrate/20180508162616_create_builds.rb]: Should we use `created_at` instead? Or do we want to stick with the current naming convention?

Comment (pytlesk4) [reply db/migrate/20180508162616_create_builds.rb]: Going to update it to created_at

Comment (pytlesk4) [reply db/migrate/20180508162616_create_builds.rb]: Also added an updated_at property

Comment (marbemac) [reply db/migrate/20180508162616_create_builds.rb]: Builds are immutable, but whatever can't hurt.

**Comment by pytlesk4** [comment]
Migrated successfully

**Pull request closed without merging**